mev-relay-rs/src/relay.rs

```diff
@@ -448,6 +451,28 @@ impl Relay {
         state.auctions.insert(auction_request, auction_context);
         Ok(())
     }
+
+    fn verify_blinded_block_signature(
+        &self,
+        signed_block: &mut SignedBlindedBeaconBlock,
+    ) -> Result<(), ethereum_consensus::Error> {
+        let proposer = self
+            .validator_registry
+            .get_public_key(signed_block.message().proposer_index())
+            .unwrap();
+        let slot = signed_block.message().slot(); // signed_block.message().slot() + 1 ???;
+        let genesis_validators_root = [0u8; 4].hash_tree_root().map_err(ConsensusError::from)?; // Should `compute_consenus_signing_root()` take Option<&Root> ?
+        let mut block =
+            signed_block.message_mut().hash_tree_root().map_err(ConsensusError::from)?;
+        let signing_root = compute_consensus_signing_root(
+            &mut block,
+            slot,
+            &genesis_validators_root,
+            &self.context,
+        )?;
+
+        verify_signature(&proposer, signing_root.deref(), signed_block.signature())
```

Comment:
should just be able to do this to get the underlying bytes

```suggestion
        verify_signature(&proposer, signing_root.as_ref(), signed_block.signature())
```